XenModel/FriendlyNames.resx

```diff
@@ -1211,6 +1211,24 @@
   <data name="Message.name-alarm" xml:space="preserve">
     <value>Unrecognized alarm</value>
   </data>
+  <data name="Message.name-vm_migrated" xml:space="preserve">

```

Comment:
Judging by other existing entries `Message.body-vm...`, I think we also need friendly versions with place holders for the message body. For those that have snapshots it would be good if we could parse the body to get the snapshot name, although the format is not very developer friendly (normally for cases like this we ask toolstack to provide the message body in xml, json would be another good idea; if it's too problematic/unreliable we could have the format changed on the xapi side).

Reply:
Currently, this is the body of the message as returned by toolstack:

![image](https://user-images.githubusercontent.com/32554698/147821588-ccbd51d2-9c6b-4c4d-a89c-b2bdbc63fbc9.png)


Could you guide me a bit on how you want me to reword/reformat them? I felt like they were quite well done.

FYI not sure how to replicate the migration one. The alert does not appear after/during migration, in either of the hosts involved.

Reply:
I'm adding the `under review` label until this point is resolved

Reply:
Indeed the friendly names have a tendency to avoid showing the uuid on the UI. But I suppose it doesn't hurt to leave it in even if it doesn't match other friendly names. I'm thinking now it's probably worse to try parse a body that might change without warning in future. Ok, ignore this.